packages/libraries/apollo/src/version.ts

```diff
@@ -1 +1 @@
-export const version = '0.36.2';
+export const version = '0.36.3';
```

Comment:
not sure why these auto bumped...

Reply:
yeah it happens when you `build`, I think it's ok to ignore or commit at this point. 
@n1ru4l @kamilkisiela  should we ignore this file?